packages/upset/src/components/Sidebar.tsx

```diff
@@ -203,7 +203,10 @@ export const Sidebar = () => {
             value={minVisible}
             onChange={ev => {
               let val = parseInt(ev.target.value, 10);
-              if (val < 0) val = 0;
+              if (Number.isNaN(val) || val < 0) {
+                val = 0;
+              }
+              ev.target.value = `${val}`;
```

Comment:
Why do we need this last line here? Isn't this already changed, triggering the `onChange`?